grblas/mask.py

```diff
@@ -0,0 +1,63 @@
+class Mask:
+    complement = False
+    structure = False
+    value = False
+
+    def __init__(self, mask):
+        self.mask = mask
+
+    def __repr__(self):
+        return f'{self.__class__.__name__} of {self.mask}'
+
+
+class ComplementedMask(Mask):
+    complement = True
+    structure = False
+    value = False
```

Comment:
No need to set the `False` values here, or no need to set all the attributes to `False` in the `Mask` base class.

Reply:
I know, but I wanted to make it easier to see what each type of mask will do.

Reply:
At a glance, I would find it easier to read if only the `True` values were specified.  Your call.